src/skeleplex/graph/image_to_graph_efficient.py

```diff
@@ -0,0 +1,67 @@
+"""Conversion of a skeletonized image into a graph representation."""
```

Comment:
```suggestion
"""Conversion of a skeletonized image into a graph representation chunk-wise.

These functions are useful for converting images that are too large to fit in memory.
"""
```

Can you also please rename the file to `image_to_graph_chunked.py`